src/swagger-specs/segmentation.yaml

```diff
@@ -3041,16 +3020,20 @@ components:
             sandboxId:
               type: string
               example: 6ed34f6f-fe21-4a30-934f-6ffe21fa3075
+              description: The ID of the sandbox.
             sandboxName:
               type: string
               example: prod
+              description: The name of the sandbox.
             type:
               type: string
               example: production
+              description: The type of the sandbox.
             default:
               type: boolean
               example: true
-          description: Details about the sandbox the audience belongs to.
+              description: A boolean value that states whether or not this sandbox is the default sandbox.
+          description: Details about the sandbox the audience belongs to. More information about sandboxes can be found in the [sandbox overview]((https://adobe.com/go/sandbox-overview-en)).
```

Comment:
Took me a moment to realize this is the description for the parent `sandbox` object and not a double description property. Can we move up to just after 3017 (i.e. before we list the object's subproperties), so it's clear which property this applies to?